learning_assistant/toggles.py

```diff
@@ -0,0 +1,28 @@
+"""
+Toggles for learning-assistant app.
+"""
+try:
+    from openedx.core.djangoapps.waffle_utils import CourseWaffleFlag
+except ImportError:
+    # mock out flag to avoid import errors during testing
+    class Flag:
```

Comment:
Pytest complains if this is not mocked out, as it expects a callable object for CourseWaffleFlag. Because the waffle flag is evaluated at run time, this needs to be mocked in some way.

Reply:
Does [how ai-aside does it](https://github.com/edx/ai-aside/blob/b2d06406041c09d27ada9dbbd210162fd78d5562/ai_aside/waffle.py) work for you?

Reply:
Yes, that's exactly what I was looking for. Thank you!